package.json

```diff
@@ -54,9 +54,9 @@
     "access": "public"
   },
   "dependencies": {
-    "@chromaui/rrweb-snapshot": "2.0.0-alpha.7-noAbsolute.2",
     "@storybook/csf": "^0.1.0",
-    "lodash.debounce": "^4.0.8"
+    "lodash.debounce": "^4.0.8",
+    "rrweb-snapshot": "^2.0.0-alpha.4"
```

Comment:
Just double-checking -- does the non-alpha `2.0.0` version of `rrweb-snapshot` work for us? Just since it's a more-stable version presumably

Reply:
Good question! `2.0.0-alpha.4` is the version they have published as `latest` on [npm](https://www.npmjs.com/package/rrweb?activeTab=versions) even though there are newer alpha versions, and there is no non-alpha 2.0. So it seems like this is the stable one of the bunch?

Reply:
Although there are many more downloads of the latest `2.0.0-alpha.11` than the `.4` version. Maybe we should use that one?

Reply:
Sounds good to me